config-linux.md

```diff
@@ -522,6 +522,8 @@ The following parameters can be specified for the container:
 
     * If `closID` is set, `l3CacheSchema` and/or `memBwSchema` is set, runtimes MUST compare `l3CacheSchema` and/or `memBwSchema` value with `schemata` file, and [generate an error](runtime.md#errors) if doesn't match.
 
+    * If `closID` is set, and neither of `l3CacheSchema` and `memBwSchema` are set, runtime MUST check if corresponding pre-configured directory `closID` present in mounted `resctrl`. If such pre-configured directory `closID` exists, runtime MUST assign container to this `closID` and [generate an error](runtime.md#errors) if directory does not exist.
```

Comment:
I think "if corresponding pre-configured directory `closID` present" is intended to be "if corresponding pre-configured directory `closID` *is* present" ?

(and possibly "if *the* directory does not exist" at the end?)

Reply:
@tianon thanks for suggestion! Merged and updated branch.